scRNA-seq-advanced/03-dataset_integration.Rmd

```diff
@@ -445,37 +612,68 @@ scater::plotReducedDim(merged_sce,
+  ggtitle("UMAP after integration with fastMNN") +
+  facet_wrap(~sample)
+```
+
+
 Importantly, one reason that batches may still appear separated in the corrected UMAP is if they _should_ be separated - for example, maybe two batches contain very different cell types.
 In this case, we would not expect batches to mix heavily, since their cell types should retain some structure. 
 Recall from earlier, we conveniently have cell type annotations in our SCEs, so we can explore this here!
-We can re-plot this UMAP to highlight cell types: A well-integrated dataset will show cell types grouped together.
-
+Let's re-plot this UMAP to highlight cell types:
 
 
 ```{r fastmnn umap celltypes}
 scater::plotReducedDim(merged_sce,
-                       # plot the fastMNN coordinates
                        dimred = "fastMNN_UMAP",
                        # color by broad celltypes
                        colour_by = "celltype_broad",
-                       # Some styling to help us see the points:
                        point_size = 0.5,
-                       point_alpha = 0.2, 
-                       # This argument will allow us to easily facet by `sample`
+                       point_alpha = 0.2) +
+  guides(colour = guide_legend(override.aes = list(size = 3, alpha = 1))) +
+  ggtitle("UMAP after integration with fastMNN")
+```
+
+Something we can see already from this UMAP is that the non-tumor cell types (mostly vascular endothelium, muscle cells, and monocytes) tend to cluster together and are generally far separated from the tumor cell types, which is an encouraging pattern!
+
+```{r fastmnn umap celltypes faceted}
+scater::plotReducedDim(merged_sce,
+                       dimred = "fastMNN_UMAP",
+                       # color by broad celltypes
+                       colour_by = "celltype_broad",
+                       point_size = 0.5,
+                       point_alpha = 0.2,
                        other_fields = "sample") +
-  ggplot2::ggtitle("UMAP after integration with fastMNN") +
-  # Modify the ggplot2 guides so that the legend points are larger and easier to see:
-  ggplot2::guides(colour = ggplot2::guide_legend(override.aes = list(size = 3, alpha = 1))) +
-  # Again, let's facet by sample so we can see this information too:
-  ggplot2::facet_wrap(~sample, nrow=2)
+  guides(colour = guide_legend(override.aes = list(size = 3, alpha = 1))) +
+  ggtitle("UMAP after integration with fastMNN") + 
+  facet_wrap(~sample)
 ```
+Looking at the cell type annotation, it appears that `SCPCL000478` uniquely is dominated by the `Tumor` cell type compared to the other samples.
+`SCPCL000479` and `SCPCL000480`, which are derived from the same patient, appears both dominated by `Tumor_Myoblast` and `Tumor_Mesoderm`, and `SCPCL000481` appears dominated by `Tumor_Myoblast` and `Tumor_Myocyte`. 
+The faceted plot also shows us that `SCPCL000478` fibroblasts appear to cluster with `SCPCL000480` and `SCPCL000481` muscle cells.
```

Comment:
So looking at these plots make me wonder if we should actually not include 478... The fact that it only contains `Tumor` cells means that those cells weren't able to be further classified into sub types. All cells were first labeled as tumor vs. non-tumor which is anything before the `_` and then if there is an additional type appended to it then the tumor cells were further stratified. But that wasn't the case for that sample. It might be worth a switch to use one of the ones that I'm using for DE that does contain the stratified tumor cells instead of 478? Or just stick with three.

Reply:
> The fact that it only contains `Tumor` cells means that those cells weren't able to be further classified into sub types.

Ahhh this sounds like something I maybe used to know! I'd definitely prefer to use a more classified SCE. I'll try something out.